static/app/views/performance/transactionSummary/relatedIssues.tsx

```diff
@@ -103,6 +103,11 @@ class RelatedIssues extends React.Component<Props> {
       query: queryParams,
     };
 
+    // Filter out key_transaction from being passed to issues as it will cause an error.
+    const conditions = tokenizeSearch(queryParams.query);
+    conditions.removeTag('key_transaction');
+    queryParams.query = stringifyQueryObject(conditions);
```

Comment:
@k-fish this should be performed in the `getIssuesEndpoint()` method of this component.

Reply:
Oh nice, good call